runtime/proto/service/log_replication.proto

```diff
@@ -99,3 +100,8 @@ message ReplicationSubscriber {
   ReplicationModel model = 1;
   string client_name = 2;
 }
+
+
+message SubscribeToReplicationMsg {
+  LogReplicationSession session = 1;
```

Comment:
Why does it carry a session object? isn't the same Channel used for many sessions? If true, then I could be multiple sessions, is it?